changelog/README.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 28th June 2023
+
+* Extended [Get all enterprises](../operations/enterprises.md#get-all-enterprises) with `LinkedUtc` filter.
```

Comment:
Maybe it could be a single entry in changelog? Extended [the endpoint] filters and response with [the property]. In any case, I think response has properties, not parameters.